database/migrations/1673447895985_users.ts

```diff
@@ -0,0 +1,27 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'profiles'
```

Comment:
If this file represents profile table then why the filename is `users.ts`?

Also, where is the migration file for users table?